src/mixer/playermanager.cpp

```diff
@@ -65,7 +65,8 @@ PlayerManager::PlayerManager(UserSettingsPointer pConfig,
                 ConfigKey("[Master]", "num_microphones"), true, true)),
         m_pCONumAuxiliaries(new ControlObject(
                 ConfigKey("[Master]", "num_auxiliaries"), true, true)),
-        m_pTrackAnalysisScheduler(TrackAnalysisScheduler::NullPointer()) {
+        m_pTrackAnalysisScheduler(TrackAnalysisScheduler::NullPointer()),
+        m_pAutoDjEnabled(std::make_unique<ControlProxy>("[AutoDJ]", "enabled", this)) {
```

Comment:
This ControlProxy is already owned by the QtObject tree. so a parented_ptr is the right smart pointer here.